### DISCUSSION TITLE
Higher multipliers for certain combinations of mods

### DISCUSSION THREAD

**Discussion by phoebethetransgirl** [ppy/osu#19253] [Ideas] [answered]
For example, random and flashlight,simply because you cant memorize a randomized map,but if you do well anyway,it would be worthwhile to reward such a performant play! maybe 1.16 instead of 1.12, to keep multipliers within range of other mod combos and to not cause a OP situation that later needs nerfing!

EDIT: Just found out you can set a arbitrary seed,forget i posted this lol

**Comment by mk56-spn** [comment, marked as answer]
random is a seeded mod, meaning even though it randomises, you can have it randomize to the exact map you wish to produce, as such theres no suprise factor

**Reply by phoebethetransgirl** [reply]
didnt realize there was a way to set a custom seed,this idea is now irrelevant
